src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstNum) => {
+    return (secondNum) => {
+      count++;
+
+      if (count > 3 && (count % 2 === 0)) {
```

Comment:
```suggestion
      if (count > 3 && count % 2 === 0) {
```
Use parentheses either in both of your expressions or don't use them at all. Stick to the one style of formatting